proto/sample_v2_round_trip.json

```diff
@@ -1100,7 +1108,7 @@
                         "name": "WordPress"
                     }
                 ],
-                "summary": {},
+                "summary": null,
```

Comment:
Does this need to be fixed seperately? Not handled by #8605?

Reply:
> Not handled by #8605?

Nope

> Does this need to be fixed seperately?

You tell me :)

Reply:
Just did it on master, same thing.  I'm not 100% sure why.  It is converting `{}` into `null` inside of detail elements.  I'm not overly concerned with it, since its a conversion difference within `details`.  I can look more into it.  I think it has to do with #8605 somehow tho 🤔

Reply:
Found it. In #8539 it was [changed](https://github.com/GoogleChrome/lighthouse/commit/5aec0639e6bdee6eca39fbec8ec5a7786f7601e5#diff-eaffc3a33b9584cf049b4d727638e15aR1103) (I think in error) and I think it is supposed to be `null`.

Reply:
@exterkamp maybe we should assert on proto `--version` in compile-proto.

Reply:
> Found it. In #8539 it was changed (I think in error) and I think it is supposed to be `null`.

nice catch. One issue is that [the `details` roundtrip check is using `toMatchObject`](https://github.com/GoogleChrome/lighthouse/blob/ec08e7b9375d3cbe38938b95a0dadfd8287d532e/lighthouse-core/test/report/proto-test.js#L36-L39), which while also helpful here, will also happily assert true for `expect({summary: null}).toMatchObject({summary: {}})` *or* `expect({summary:{}}).toMatchObject({summary: {}})`

Is it supposed to be `null`, though? It's `{}` in `sample_v2.json`

Reply:
after #8867, these will need to be `{}`. Update to > 3.7.1 to get that automatically :)

Reply:
I upgraded to 3.7.1 and its still `null`...